platform/api.annotations.common/build.xml

```diff
@@ -24,79 +24,4 @@
 
     <import file="../../nbbuild/templates/projectized.xml"/>
 
-    <!--
-    <target name="findbugs-annotations-init" if="api.annotations.common.findbugs">
-        <echo>Building common annotations for usage with FindBugs...</echo>
-
-        <property name="src.dir" location="build/generated"/>
-        <property name="src.common" location="src"/>
-
-    </target>
-
-    <target name="build-init" depends="findbugs-annotations-init, projectized.build-init"/>
-
-    <target name="compile" depends="findbugs-annotations, projectized-common.compile"/>
-
-    <target name="findbugs-annotations" if="api.annotations.common.findbugs" depends="build-init, projectized.findbugs-init">
```

Comment:
It's been a long time so I don't remember exactly (and I'm not sure where this might have evolved) but this was present to change annotations to class retention and thus usable in FindBugs/SpotBugs check as it operates on class files.

Reply:
Welcome back Petr! Looks like the annotations now have `Retention.CLASS` by default: https://bits.netbeans.org/12.3/javadoc/org-netbeans-api-annotations-common/org/netbeans/api/annotations/common/package-summary.html

Otherwise `javac` wouldn't see them when `org-netbeans-api-annotation-commons.jar` is used as a library (which produces the nasty warnings). But probably I should have apply `Retention.CLASS` to `Nonnull`, `TypeQualifierNickname` and `TypeQualifier` as well...

Reply:
Ok, then. Thanks @JaroslavTulach. I guess it would be safer to have the other annotations with CLASS retention as well.